### PR TITLE
WDAC/AppLocker: URL corrections

### PR DESCRIPTION
**Pull request by RAJU2529**
As per the user report #6782.  so i removed two invalid links and then added two new links.
below are the new links
**https://docs.microsoft.com/windows/security/threat-protection/windows-defender-application-control/wdac-and-applocker-overview**
**https://docs.microsoft.com/windows/security/threat-protection/windows-defender-application-control/feature-availability**

### CODE REVIEW

File: windows/security/threat-protection/windows-defender-application-control/windows-defender-application-control.md [hunk 1]
```diff
@@ -45,8 +45,8 @@ Windows 10 includes two technologies that can be used for application control de
 
 | Topic | Description |
 | - | - |
-| [WDAC and AppLocker Overview](plan-windows-defender-application-control-management.md) | This topic describes the decisions you need to make to establish the processes for managing and maintaining WDAC policies. |
-| [WDAC and AppLocker Feature Availability](understand-windows-defender-application-control-policy-design-decisions.md) | This topic lists the design questions, possible answers, and ramifications of the decisions when you plan a deployment of application control policies. |
+| [WDAC and AppLocker Overview](https://docs.microsoft.com/windows/security/threat-protection/windows-defender-application-control/wdac-and-applocker-overview) | This topic describes the decisions you need to make to establish the processes for managing and maintaining WDAC policies. |
+| [WDAC and AppLocker Feature Availability](https://docs.microsoft.com/windows/security/threat-protection/windows-defender-application-control/feature-availability) | This topic lists the design questions, possible answers, and ramifications of the decisions when you plan a deployment of application control policies. |
```
Comment (illfated) [inline windows/security/threat-protection/windows-defender-application-control/windows-defender-application-control.md:49]: Please, if possible, keep the links to GitHub, unless the Microsoft Docs team require us to stop using relative links.
```suggestion
| [WDAC and AppLocker Overview](wdac-and-applocker-overview.md) | This topic describes the decisions you need to make to establish the processes for managing and maintaining WDAC policies. |
| [WDAC and AppLocker Feature Availability](feature-availability.md) | This topic lists the design questions, possible answers, and ramifications of the decisions when you plan a deployment of application control policies. |
```

Comment (illfated) [reply windows/security/threat-protection/windows-defender-application-control/windows-defender-application-control.md:49]: Note that these links are the same pages, only GitHub pages, as long as the source page is the GitHub version of the page. These links are automatically translated when migrated to the docs.microsoft.com pages.

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

**Comment by illfated** [comment]
@RAJU2529 : Kindly commit my suggested URL version (your original page correction is correct, although using the external URLs. I only recommend using the GitHub version locally).

**Comment by illfated** [comment]
Cosmetic title improvement suggestion: **WDAC/AppLocker: URL corrections**

**Comment by RAJU2529** [comment]
@illfated. I understood that both are same md page,but I pointed to particular topic page url , so I changed it

**Comment by illfated** [comment]
(Thank you for the PR title change, by the way.)

Fair enough, but I am suggesting the very same page, just the GitHub version of the same page as you have entered.

URL correction 1:
- incorrect URL: plan-windows-defender-application-control-management.md (https://github.com/MicrosoftDocs/windows-itpro-docs/blob/public/windows/security/threat-protection/windows-defender-application-control/plan-windows-defender-application-control-management.md)
- your URL correction: https://docs.microsoft.com/windows/security/threat-protection/windows-defender-application-control/wdac-and-applocker-overview
- my URL suggestion: wdac-and-applocker-overview.md (https://github.com/MicrosoftDocs/windows-itpro-docs/blob/public/windows/security/threat-protection/windows-defender-application-control/wdac-and-applocker-overview.md) (same page as yours, just keeping it to GitHub, as in the original page format)

URL correction 2:
- incorrect URL: understand-windows-defender-application-control-policy-design-decisions.md (https://github.com/MicrosoftDocs/windows-itpro-docs/blob/public/windows/security/threat-protection/windows-defender-application-control/understand-windows-defender-application-control-policy-design-decisions.md)
- your URL correction: https://docs.microsoft.com/windows/security/threat-protection/windows-defender-application-control/feature-availability
- my URL suggestion: feature-availability.md (https://github.com/MicrosoftDocs/windows-itpro-docs/blob/public/windows/security/threat-protection/windows-defender-application-control/feature-availability.md)

Notice that I am trying to keep the original URL format (page here on GitHub), while still using the same page as yours, just the GitHub version of the page.

Your URL 1: https://docs.microsoft.com/windows/security/threat-protection/windows-defender-application-control/wdac-and-applocker-overview
My URL 1: https://github.com/MicrosoftDocs/windows-itpro-docs/blob/public/windows/security/threat-protection/windows-defender-application-control/wdac-and-applocker-overview.md

Your URL 2: https://docs.microsoft.com/windows/security/threat-protection/windows-defender-application-control/feature-availability
My URL 2: https://github.com/MicrosoftDocs/windows-itpro-docs/blob/public/windows/security/threat-protection/windows-defender-application-control/feature-availability.md

In addition to keeping the URL format the same as the original GitHub page, my suggestion also makes it easier to browse between the linked GitHub pages, the same way as the old version does.

**Comment by RAJU2529** [comment]
@illfated . in commit  suggestions your only  **wdac-and-applocker-overview.md** 
and  **feature-availability.md**   are available, what about  remaining  full path of the url

**Comment by illfated** [comment]
I know. That is because they are relative links, residing in the same directory as this document page. That is why it works (and why the incorrect links also work) without the full path. I only included the full GitHub path to make it easier for you to click the links and compare the contents.

By the way, thank you for committing my suggestions.

**Comment by illfated** [comment]
#sign-off

**Comment by mypil** [comment]
@denisebmsft - PR has been copy edited and is ready for final review. This is based on issue #6782. 

Thank you.

cc: @kenwith

**Pull request merged**